xlsx/tests/test.rs

```diff
@@ -399,3 +399,37 @@ fn test_exporting_merged_cells() {
 
     fs::remove_file(temp_file_name).unwrap();
 }
+
+#[test]
+fn test_merged_cell_behaviors() {
+    // loading the xlsx file containing merged cells
+    let example_file_name = "tests/merged_cells.xlsx";
+    let mut model = load_from_xlsx(example_file_name, "en", "UTC").unwrap();
+
+    model.set_user_input(0, 1, 5, "Hello".to_string());
+
+    // making sure that merge cell should not get updated
+    assert_ne!(model.get_cell_content(0, 1, 5).unwrap(), "Hello");
+    assert_ne!(model.get_cell_type(0, 1, 5).unwrap(), CellType::Text);
+
+    model.update_cell_with_bool(0, 1, 5, true);
+    assert_ne!(model.get_cell_content(0, 1, 5).unwrap(), "true");
```

Comment:
Ha! I didn't know about the `assert_ne` macro! TIL.
But wouldn't it be easier to check for equality?

Reply:
Yes, I can check for equality as well. Introduced negation here as its more intuitive, we just want to test that update_cell_with_bool does not set the cell as true as its a part of merge cell.

Reply:
Added equality assertion instead of negation